dll/kernel32.cpp

```diff
@@ -1368,17 +1379,19 @@ namespace kernel32 {
 	}
 
 	unsigned int WIN_FUNC GetCurrentDirectoryA(unsigned int uSize, char *lpBuffer) {
-		DEBUG_LOG("GetCurrentDirectoryA(%u, %p)\n", uSize, lpBuffer);
+		DEBUG_LOG("GetCurrentDirectoryA(%u, %p)", uSize, lpBuffer);
```

Comment:
Why remove the \n?

Reply:
So the debug logs I added get printed in the same line